packages/e2e-test-utils-playwright/src/editor/modify-post-metadata.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Clicks on the button in the header which opens Document Settings sidebar when
```

Comment:
I don't think this description is correct.

Reply:
I'm wondering if it is okay to create a new global util or start with something specific for this tests. Something like the [UndoUtils](https://github.com/WordPress/gutenberg/blob/804761ce837eb70b96314b4e12d8fb7fe301dfbe/test/e2e/specs/editor/various/undo.spec.js#L502-L533), for example. (There are many other tests with their own utils)

Reply:
I updated the description and moved the test 👍